example/cycle/test/test.js

```diff
@@ -7,31 +7,25 @@ import * as actions from '../../actions';
 
 import { fetchReposByUser, searchUsers } from '../';
 
-function assertSourcesSinks(sources, sinks, main, done) {
-  const Time = mockTimeSource();
+function assertSourcesSinks(sources, sinks, main, done, timeOpts = {}) {
```

Comment:
Even if we did that, there's still the isolation implementation of the DOM driver to consider. Personally I think we should stick with `mockDOMSource`. The usage is [quite nice](https://github.com/cyclejs/cyclejs/tree/master/dom#-mockdomsourcemockconfig).

Since isolation semantics vary from driver to driver I think it makes more sense for each driver to provide a `mockDriverNameSource` if it is non-trivial to mock the implementation.

Perhaps what we're really missing here is a nice `mockHTTPSource`.